modules/dag-l1/src/test/scala/org/tessellation/dag/l1/infrastructure/DatabaseSuite.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.dag.l1.infrastructure.db
+
+import cats.effect.IO
+
+import org.tessellation.dag.l1.config.types.DBConfig
+import org.tessellation.dag.l1.infrastructure.address.storage.AddressStorage
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Balance
+
+import ciris.Secret
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.all.NonNegLong
+import weaver.SimpleIOSuite
+
+object DatabaseSuite extends SimpleIOSuite {
```

Comment:
Thank you for trying to write a unit tests, however database in such form is deprecated in the project and we would like to remove it anyway :D We will use in-memory storage for temporary (quick-changing) data and global snapshots are stored directly on disk.

Reply:
Upon discussing this with @ryle-goehausen , this PR is being closed.